libs/features/components/banners/src/lib/components/team-member/team-member.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-team-member',
+  templateUrl: './team-member.component.html',
+  styleUrls: ['./team-member.component.scss'],
+})
+export class TeamMemberComponent {
+  @Input () name = 'Jente Rosseel'
+  @Input () title = 'Founder,Managing Partner'
+  @Input () imageSrc = 'https://i.postimg.cc/6pWDZNNp/jente.jpg'
+  @Input () descrition = 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero quia quae aspernatur similique, adipisci, debitis hic commodi impedit natus exercitationem tempore error, ratione dolore aliquam quis. Officia officiis veniam commodi.'
+  @Input () linkedinUrl = ''
```

Comment:
Please take note that this component is a carousel too as depicted on the design hence takes an array of the data above